### ISSUE TITLE
Create sample data for OCDS 1.1?

### ISSUE THREAD

**Issue by dc-coder** [open-contracting/sample-data#26]
Is sample data or templates for OCDS 1.1 on the to do list?

**Comment by timgdavies** [comment]
There is some work in progress on fictional 1.1 data on the [1.1 branch](https://github.com/open-contracting/sample-data/tree/1.1/fictional-example) and some draft work there also on an [upgrade script](https://github.com/open-contracting/sample-data/blob/1.1/fictional-example/update.py) from 1.0 -> 1.1

We're hoping to get the repository updated to be 1.1 data over the coming week. Pull requests with updated samples/examples always welcome - as are ideas on which the most useful sample data is.

**Comment by dc-coder** [comment]
Thanks for update. I will def try to share some sample data and i'll try to be more collaborative and submit some pull requests.

For the second part of your comment on ideas about which sample data are most useful:

- Sample data preferably in **csv** format that shows the full worked example of a stage or a building block... for me, sample data is just an example that may or may not cover the full schema... having the full worked example in csv is important for the non-technical audience who are not familiar with json schema. If you are a developer, you can figure out the schema... and you are used to reading json. (even tho sample data is still handy, because reading the manual is not as fun)

But for someone in an agency or for example a procurement specialist... it would be great help, if instead of giving a link to a standard documentation, if they can see worked example, with actual data that makes things "sink in"... it can be really useful and then they can figure out what data a specific organization has or what type of use cases can be done.

I think of  specific building blocks or just one stage as useful because it is a logical grouping and less overwhelming, if someone is working with a block and needs to see how it actually looks like.... it is easy to find the corresponding block. If someone is working on a part of the process... (planning, tender, award, contract, implementation) - it is useful if they can see a worked example of the specific stage.... 

The full example is important, because then people don't need to cross check with schema.

I hope this is useful.

**Comment by duncandewhurst** [comment]
@dc-coder thanks for the thoughts - very useful.

I've made some updates the [1.1 sample data](https://github.com/open-contracting/sample-data/tree/1.1/fictional-example) which hopefully address some of your points:

* There are JSON examples of releases from each stage of a contracting process, including all of the changed and new fields in 1.1. These should include fully completed examples of all building blocks and sections (except some examples where building blocks are repeated, e.g. milestones aren't provided in every section)

* There are [XLSX versions](https://github.com/open-contracting/sample-data/tree/1.1/fictional-example/flattened) of each release - based on the structure of the input template we designed for the PPP project.

It would be great to get your feedback on whether these meet your needs and suggestions/pull requests for improvements.

Thanks!
